test/PrintAsObjC/classes.swift

```diff
@@ -318,6 +318,13 @@ typealias AliasForNSRect = NSRect
   func testBridgingOptionality(_ a: UnsafePointer<Int>?, b: UnsafeMutablePointer<Int>!, c: AutoreleasingUnsafeMutablePointer<Methods?>?) {}
 }
 
+// CHECK-LABEL: IB_DESIGNABLE
+// CHECK-LABEL: @interface MyDesignableObject : NSObject
```

Comment:
Here `CHECK-LABEL` is a sort of "start over" annotation. I think it makes more sense to check that the `IB_DESIGNABLE` is actually attached to MyDesignableObject using CHECK-NEXT, even though it means we need to match the next line as well.